_posts/phuong-my_do/2017-04-28-Business_analyst.md

```diff
@@ -0,0 +1,112 @@
+
+### The Quality Assurance Team 
+The QAs perform Quality Assurance based on requirements written by Brian.
+
+### The users (bioinformaticians, researchers, students, the cancer research community). 
+
+Brian will provide assistance in training the users to their new application. For larger projects a user service team could be dedicated; in that case, Brian would provide Level 2 support to the user service Team for a better understanding of the solution.
+   
+**The key in such IT projects is for Brian not only to document but mostly to understand** what bioinformaticians, researchers and software engineers tell him to accurately document it. 
+It’s about asking the right question. To be able to do so, Brian needs to understand the business goals as much as the technical aspects of an application.
+
+Who has never seen this scenario? User Service Team: “The user cannot use the feature, is it a development issue? >> Developper: "No, it’s a data issue" >> Data Quality: "No, it’s an environment or network issue" >> Dev-Ops: "No, the user did not use it properly”. Brian’s role here is to clarify the issue, record everyone’s understanding and challenge the answers to ensure the right solution is identified.
+
+# Brian’s techniques and main activities
+* **Define the 5 W’s**: Who did What, Where, When, and Why: Probably the basics when Brian starts business analysis. Answering What and Why is sometimes challenging in the cancer genomics research field characterized by exploration (“I don’t know exactly what I want but we should keep moving in this direction”)
+
+* **Analyse the existing system and processes, benchmark and best practices**: there is a lot of reading in a Brian’s job, all the documentation of what your company and its competitors produce.    
+
+* **Lead brainstorming sessions, interviews and workshops, facilitate meetings**: a project needs someone “not biased” to make sure everyone’s opinion from business and technical perspective will be heard, understood and recorded.
+
+* **Create documentation**: business processes and rules, data flow diagrams, mock-ups and wireframes, user stories, user acceptance criteria, functional and non-functional requirements, decision logs, scope and rationale of a project, user impact of any business and technical decisions. 50% of Brian’s job is to write documentation and the other 50% is to listen to and communicate with people to make sure the documentation is accurate and shared.
+
+* **Review documentation** with the business stakeholders.
+
+* **Participate in solution design** (data modeling) with the software engineering team and organize walkthrough of the specifications with developers. 
```

Comment:
walkthrough is weird here.

maybe just:

software engineering team and organize the specifications with developers.